public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-loader.php

```diff
@@ -251,6 +249,21 @@ public static function get_active_wordcamp_statuses() {
 		);
 	}
 
+	/**
+	 * Get all the status that occur after a camp has a signed contract.
+	 *
+	 * @return array
+	 */
+	public static function get_after_contract_statuses() {
+		return array_merge(
+			array(
+				'wcpt-needs-fill-list',
+				'wcpt-needs-schedule',
+			),
+			self::get_public_post_statuses()
```

Comment:
hmmm, it seems very unlikely that we'd have a future status that is public but before the contract is signed, unless something dramatic changes about the application workflow.

There could also be a problem if we hardcode it and then something is added to `get_public_post_statuses()` and not added to this function.

Either way, we should be checking all the status functions whenever a status is added/removed, so it may not be an issue. I don't feel too strongly, though.

Reply:
Yeah, that's a fair point. I'll change it to be hardcoded.

Reply:
fixed in 66ee8b46e6936ea60ad41a5d7b8ecc455f76f9b2